### PR TITLE
re-arrange fields to match key

### PR DESCRIPTION
**Pull request by brmcdoug**

### CODE REVIEW

File: pkg/arangodb/edge.go [hunk 1]
```diff
@@ -86,11 +86,16 @@ func (a *arangoDB) lsNodeHandler(obj *notifier.EventMessage) error {
...
-	Link string `json:"link"`
+	Key        string       `json:"_key"`
+	From       string       `json:"_from"`
+	To         string       `json:"_to"`
+	Link       string       `json:"link"`
+	ProtocolID base.ProtoID `json:"protocol_id"`
+	DomainID   int64        `json:"domain_id"`
+	MTID       uint16       `json:"mt_id"`
+	AreaID     string       `json:"area_id"`
+	FromLSNode string       `json:"from_lsnode"`
```
Comment (sbezverk) [inline pkg/arangodb/edge.go:97]: Line 97 and Link 98 look duplication to me, what is the value to have the same info?

File: pkg/arangodb/edge.go [hunk 2]
```diff
@@ -174,11 +179,16 @@ func (a *arangoDB) processEdge(ctx context.Context, key string, e *message.LSLin
...
-		Link: e.Key,
+		Key:        key,
+		From:       ln.ID,
+		To:         rn.ID,
+		Link:       e.Key,
+		ProtocolID: e.ProtocolID,
+		DomainID:   e.DomainID,
+		MTID:       uint16(mtid),
+		AreaID:     e.AreaID,
+		FromLSNode: ln.Name,
```
Comment (sbezverk) [inline pkg/arangodb/edge.go:190]: Same comment as above

**Review by sbezverk — CHANGES_REQUESTED** [review]

File: pkg/arangodb/edge.go (hunk 1 above)
Comment (brmcdoug) [reply pkg/arangodb/edge.go]: good point.  i was experimenting with adding node name, but this info can easily be found in LSNode collection

File: pkg/arangodb/edge.go (hunk 2 above)
Comment (brmcdoug) [reply pkg/arangodb/edge.go]: same reply.  removed in latest commit

**Review by sbezverk — APPROVED** [review]

**Pull request merged**